dotCMS/src/main/java/com/dotmarketing/fixtask/tasks/FixTask00085FixEmptyParentPathOnIdentifier.java

```diff
@@ -0,0 +1,100 @@
+package com.dotmarketing.fixtask.tasks;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.dotmarketing.beans.FixAudit;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.db.HibernateUtil;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.fixtask.FixTask;
+import com.dotmarketing.portlets.cmsmaintenance.ajax.FixAssetsProcessStatus;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.MaintenanceUtil;
+
+public class FixTask00085FixEmptyParentPathOnIdentifier implements FixTask {
+
+    private int total = 0;
+
+    @Override
+    public List<Map<String, Object>> executeFix() throws DotDataException {
+
+        final List<Map<String, Object>> returnValue = new ArrayList<>();
+
+        Logger.info(FixTask00085FixEmptyParentPathOnIdentifier.class,
+                "Beginning FixTask00085FixEmptyParentPathOnIdentifier");
+
+        if (!FixAssetsProcessStatus.getRunning()) {
+
+            FixAssetsProcessStatus.startProgress();
+            FixAssetsProcessStatus
+                    .setDescription("Task 85: Updating Empty Parent Path in Identifiers");
+            FixAssetsProcessStatus.setTotal(total);
+
+            try {
+                HibernateUtil.startTransaction();
+                final DotConnect dc = new DotConnect();
+
+                dc.setSQL(
+                        "update identifier set parent_path = '/' where parent_path is null or parent_path = ''");
+
+                try {
+                    dc.loadResult();
+                } catch (DotDataException e) {
+                    Logger.error(this, e.getMessage(), e);
+                }
+
+                FixAssetsProcessStatus.setErrorsFixed(total);
+
+                final FixAudit audit = new FixAudit();
+                audit.setTableName("identifier");
+                audit.setDatetime(new Date());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use the Java 8 Date and Time API instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2143)